bin/bix

```diff
@@ -561,13 +562,24 @@ do_ex_test_deep() {
 
     export MIX_ENV=test
     run_mix "do" deps.get, compile --warnings-as-errors >/dev/null
```

Comment:
Is it possible to use the setup function here

Reply:
You mean like have a function that just does the mix commands to set up the database? And both `ex-test-deep` and `ex-test-setup` would call that?

Or something else?

Reply:
I don't think we even need a new function. If deep test calls the new function with the right args I think we can remove some duplicate code

Reply:
I think I'm tracking. Lemme know if this is along the lines of what you're thinkin'.